mintpy/ifgram_inversion.py

```diff
@@ -125,7 +125,7 @@ def create_parser():
     par.add_argument('--parallel', dest='parallel', action='store_true',
                      help='Enable parallel processing for the pixelwise weighted inversion.')
     par.add_argument('--cluster', '--cluster-type', dest='cluster', type=str,
-                     default='slurm', choices={'lsf', 'pbs', 'slurm'},
+                     default='local', choices={'lsf', 'pbs', 'slurm', 'local'},
```

Comment:
I am fine with changing the default cluster type if you and @falkamelung are fine with it. If you do, let's change the corresponding value and comments in smallbaselineApp.cfg and smallbaselineApp_auto.cfg as well.

Reply:
This is how Falk intends to use Dask.